uint64 timestamp	# timestamp of the setpoint
bool landed		# true if vehicle is currently landed on the ground
